install/helm/agones/templates/hooks/pre_delete_hook.yaml

```diff
@@ -47,6 +47,10 @@ spec:
         volumeMounts:
         - name: script
           mountPath: /scripts/
+      {{- if .Values.agones.controller.tolerations }}
```

Comment:
I like this approach. It's the easiest implementation. If people need to have their own custom toleration, we could provide an extra configuration for delete_job toleration, and then default to the controller toleration :+1: